docs/MO_DG/prepare_model/customize_model_optimizer/Extending_MXNet_Model_Optimizer_with_New_Primitives.md

```diff
@@ -1,45 +1,41 @@
-# Extending the MXNet Model Optimizer with New Primitives  {#openvino_docs_MO_DG_prepare_model_customize_model_optimizer_Extending_MXNet_Model_Optimizer_with_New_Primitives}
+# Extending the Model Optimizer for Custom MXNet* Operations {#openvino_docs_MO_DG_prepare_model_customize_model_optimizer_Extending_MXNet_Model_Optimizer_with_New_Primitives}
 
-This section describes how you can create a Model Optimizer extension for a custom layer from your MXNet* model. It supplements the main document [Extending Model Optimizer with New Primitives](Extending_Model_Optimizer_with_New_Primitives.md) and provides a step-by-step procedure. To create an extension for a particular layer, perform the following steps:
+This section provides instruction on how to support a custom MXNet operation (or as it called in the MXNet documentation
+"operator" or "layer") which is not a part of the MXNet operation set. For example, if the operator is implemented using
+the following [guide](https://mxnet.apache.org/versions/1.7.0/api/faq/new_op.html)).
+
+This section describes a procedure on how to extract operator attributes in the Model Optimizer. The rest of the
```

Comment:
"extract operator attributes". may be "attributes' operator" or "operator of attributes" or "operator, attributes"

Reply:
And I think you should change 'operator' to 'operation'.

Reply:
Again, we refer to the MXNet operator, not OpenVINO operation. 

The point of this sentence is that we have an operator (or operation) and we want to extract attributes of this operation. For example, for Convolution we want to extract strides and paddings etc. Both of your suggestions have different meaning.